src/components/drawer/__internal__/drawer.stories.mdx

```diff
@@ -91,6 +91,30 @@ import Drawer from 'carbon-react/lib/components/drawer';
   </Story>
 </Preview>
 
+### Custom height
+
+<Preview>
+  <Story name="custom height" parameters={{ info: { disable: true }, chromatic: { disable: true }}} >
+    return (
+      <Drawer
+        height='230px'
```

Comment:
What are you thoughts on making this a knob? Like `expandedWidth` and `animationDuration`. It would mean our consumers can change the value on this example when in the `Canvas` tab on Storybook. So they could experiment with custom heights for example.